src/herder/HerderImpl.cpp

```diff
@@ -1575,16 +1601,18 @@ HerderImpl::restoreSCPState()
     ZoneScoped;
     // setup a sufficient state that we can participate in consensus
     auto const& lcl = mLedgerManager.getLastClosedLedgerHeader();
+
+    setTrackingSCPState(lcl.header.ledgerSeq, lcl.header.scpValue, false);
```

Comment:
in this commit I would not expect to see this:
old code would just leave the app in `BOOTING` state?

Reply:
after reviewing the commit as a whole, I am not sure we need (in this commit) `HERDER_TRACKING_LCL_STATE`. If we need it, it should be introduced in a later commit (otherwise we have too many things happening in the same commit, making things hard to review).

From this commit point of view I'd expect to
1. in the "genesis" case below, just leave the state untouched (so booting?)
2. after that state is just `HERDER_TRACKING_NETWORK_STATE`